test/unit/Resource/Collection.test.js

```diff
@@ -58,4 +58,27 @@ describe('The collection', () => {
         expect(collection.isLoaded).toBeTruthy();
       });
     }));
+  it('can be cloned using the clone method.', () => {
```

Comment:
Wellicht goed om ook een diepere inspectie te doen en iets aan te passen in de clone en dan te verifieren of dit niet aangepast is in de collection. Iets by reference hebben in JS is zo makkelijk fout gedaan...

Reply:
HIer heb ik over nagedacht: mijn idee was om de properties die bij een set geheel vervangen worden niet de testen in de deep clone. Immers: het object zelf wordt in principe nooit veranderd.